src/main/scala/com/mozilla/telemetry/pings/MainPing.scala

```diff
@@ -107,6 +133,26 @@ case class MainPing(application: Application,
     dynamicProcessEvents.filter(_.category == "normandy")
   }
 
+
+  override def pingLevelEvents: Seq[JObject] = {
+    import org.json4s.JsonDSL._
+
+    val pingSent = pingAmplitudeProperties merge
+      ("session_id" -> sessionStart) ~
+      ("insert_id" -> (getClientId.getOrElse("None") + sessionStart.toString + "ping sent")) ~
+      ("event_type" -> "ping sent") ~
+      ("time" -> sessionStart) ~
```

Comment:
This should probably be the ping's created time since this "event" is at the end of a session